### ISSUE TITLE
Accessibility - Landing Page - color contrast issues with navigation bar

### ISSUE THREAD

**Issue by terrimorton** [dotnet-foundation/website#295]
Page: https://dotnetfoundation.org/

All 6 options in the navigation bar have an insufficient color contrast ratio of 4.35. WCAG 2 level AA requires a contrast ratio of 4.5:1 or greater for normal text.

Screenshot showing Accessibility Insights for Web failures:
![image](https://user-images.githubusercontent.com/4643903/89969789-481d4580-dc25-11ea-9642-27c38c81d37b.png)

Full failure details for the "Membership" menu option:

**Title**: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (#membership-dropdown\ membership-dropdown--mobile)
Tags: Accessibility, WCAG 1.4.3, color-contrast

**Issue**: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=msftAI)

**Target application**: .NET Foundation - https://dotnetfoundation.org/

**Element path**: #membership-dropdown\ membership-dropdown--mobile

**Snippet**: <a class="nav-link dropdown-toggle-custom" id="membership-dropdown membership-dropdown--mobile" href="/member" role="button" aria-haspopup="true" aria-expanded="false"> Membership </a>

**How to fix**: 
Fix any of the following:

-  Element has insufficient color contrast of 4.35 (foreground color: #797979, background color: #ffffff, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1

**Environment**: Chrome version 84.0.4147.105

====

This accessibility issue was found using Accessibility Insights for Web 2.20.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by terrimorton** [comment]
My suggestion here would be to change #797979 to #212529. This would change the menu color to match the menu dropdown color.

![image](https://user-images.githubusercontent.com/4643903/90050715-99204e80-dca4-11ea-8dee-f85221754ae6.png)


Before (#797979):
![image](https://user-images.githubusercontent.com/4643903/90050962-f1575080-dca4-11ea-83ac-3aa068f6026c.png)

After (#212529):
![image](https://user-images.githubusercontent.com/4643903/90050974-f6b49b00-dca4-11ea-90e2-702863582db7.png)

**Comment by Arhell** [comment]
@terrimorton ready on main site and membership repository too

**Comment by terrimorton** [comment]
Thank you, @Arhell, looks great and has resolved the contrast issue in Accessibility Insights!
